src/components/ImageCard/ImageHotspots.test.jsx

```diff
@@ -1,6 +1,9 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import { act, isDOMComponent } from 'react-dom/test-utils';
+import { render } from '@testing-library/react';
+import { screen } from '@testing-library/dom';
```

Comment:
Is there a reason we brought in screen through the DOM package here instead of the React package like your other test?

Reply:
No, just lazy coding on my part. Fixed.